src/Common/src/Interop/Windows/user32/Interop.GetWindowTextLengthW.cs

```diff
@@ -9,7 +9,7 @@ internal partial class Interop
 {
     internal partial class User32
     {
-        [DllImport(Libraries.User32, EntryPoint = "GetWindowTextLengthW")]
-        public static extern int GetWindowTextLength(IntPtr hWnd);
+        [DllImport(Libraries.User32, SetLastError = true, ExactSpelling = true)]
+        public static extern int GetWindowTextLengthW(IntPtr hWnd);
```

Comment:
Yeah, I think we should remove them all. It isn't a ton of space, but they can add up unnecessarily.